web/.flowconfig

```diff
@@ -4,6 +4,8 @@
 <PROJECT_ROOT>/lib-dist/.*
 .*/node_modules/.*/__tests__/.*
 .*/build-configs/.*/assets/manifest\.json
+; TODO Fix flow error
+.*/node_modules/@integreat-app/react-sticky-headroom/index\.js\.flow
```

Comment:
Seems to be our library? :) So we should fix it there

Reply:
It is. You could create an issue for this.

Reply:
Added Issue https://issues.tuerantuer.org/browse/HEADROOM-9